net/mwan3/files/usr/libexec/rpcd/mwan3

```diff
@@ -82,6 +82,7 @@ get_mwan3_status() {
 	network_get_device device $1
 
 	if [ "${iface}" = "${iface_select}" ] || [ "${iface_select}" = "" ]; then
+		#TODO use procd to see if tracker is running
```

Comment:
So that it is not forgotten. Here the pgrep has to be adapted

Reply:
Thanks. I'm working on a fix now. I just need a few days to test for potential regressions.

Reply:
@aaronjg you delete in the last push force the pid but not the pgrep command in the next line. Could you also change this please.

Reply:
done